common/build/eslint-config-fluid/strict-ts.json

```diff
@@ -0,0 +1,1440 @@
+{
+  "env": {
+    "es6": true,
+    "browser": true,
+    "node": true
+  },
+  "globals": {
+    "Atomics": "readonly",
+    "SharedArrayBuffer": "readonly"
+  },
+  "parser": "/workspaces/FluidFramework/common/build/eslint-config-fluid/node_modules/@typescript-eslint/parser/dist/index.js",
+  "parserOptions": {
+    "ecmaVersion": "latest",
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "project": "./tsconfig.json"
+  },
+  "plugins": [
+    "promise",
+    "@typescript-eslint",
+    "@rushstack/security",
+    "@rushstack",
+    "eslint-comments",
+    "import",
+    "react",
+    "tsdoc",
+    "unicorn",
+    "editorconfig"
+  ],
+  "reportUnusedDisableDirectives": true,
+  "rules": {
+    "@typescript-eslint/explicit-member-accessibility": [
+      "error",
+      {
+        "accessibility": "explicit",
+        "overrides": {
+          "accessors": "explicit",
+          "constructors": "no-public",
+          "methods": "explicit",
+          "properties": "explicit",
+          "parameterProperties": "explicit"
+        }
+      }
+    ],
+    "@typescript-eslint/no-parameter-properties": [
+      "warn",
+      {
+        "allows": [
+          "private",
+          "private readonly",
+          "public readonly",
+          "readonly"
+        ]
+      }
+    ],
```

Comment:
This prevents properties from being declared within the parameters of constructors unless they're private or readonly.

Reply:
This will probably be super disruptive to fix up, and I personally don't mind this TS language feature.  And what's the reasoning behind the specific restrictions here (only allow private/readonly)?

Reply:
Declaring parameters in constructors is foreign to non-TS devs, but more importantly, they make it more difficult to find where class-level properties are declared when reading code. It's a variant of the "declare as close to use as possible" rule -- "declare all class members together."

But I agree that cleanup may be ugly. And it does require more code.

Reply:
> And it does require more code

At least the emitted JS is the same, it appears, based on a quick check in TS Playground.

If there were a rule to require each such param on its own line in the constructor declaration I'd definitely be in favor of that, I'm strict to do that myself.